Library/Homebrew/cmd/analytics.rb

```diff
@@ -0,0 +1,44 @@
+#:  * `analytics` [`state`]:
```

Comment:
Would be good to point to the analytics documentation in here and use the language from there (anonymous user behaviour analytics)